spec/computable/registry/apply.spec.ts

```diff
@@ -0,0 +1,167 @@
+import * as ganache from 'ganache-cli'
+import Web3 from 'web3'
+import { Contract, Block } from '../../../node_modules/web3/types.d'
+import { stringToBytes, increaseTime } from '../../helpers'
+import Eip20 from '../../../src/contracts/eip-20'
+import Voting from '../../../src/contracts/plcr-voting'
+import Parameterizer from '../../../src/contracts/parameterizer'
+import Registry from '../../../src/contracts/registry'
+import { ParameterDefaults, NAME } from '../../../src/constants'
+import {
+  deployDll,
+  deployAttributeStore,
+  maybeParseInt,
+  eventReturnValues,
+} from '../../../src/helpers'
+
+const provider:any = ganache.provider(),
+  web3 = new Web3(provider)
+
+let accounts:string[],
+  eip20:Eip20,
+  dll:Contract,
+  store:Contract,
+  voting:Voting,
+  parameterizer:Parameterizer,
+  registry:Registry
+
+describe('Registry', () => {
```

Comment:
This is a really nice test suite for the registry. Has me thinking how we could put together a stress test suite, say with 100 simultaneous listing requests hitting the registry.